src/gui/qgsmaplayerstylecategoriesmodel.cpp

```diff
@@ -110,219 +110,350 @@ QVariant QgsMapLayerStyleCategoriesModel::data( const QModelIndex &index, int ro
       switch ( role )
       {
+        case static_cast< int >( Role::NameRole ):
+          return name;
         case Qt::DisplayRole:
-          return tr( "Rendering" );
+
+          return htmlStylePattern.arg( name ).arg( description );
         case Qt::ToolTipRole:
-          return tr( "Scale visibility, simplify method, opacity" );
+          return description;
         case Qt::DecorationRole:
           return QgsApplication::getThemeIcon( QStringLiteral( "/propertyicons/rendering.svg" ) );
       }
       break;
+    }
     case QgsMapLayer::StyleCategory::CustomProperties:
+    {
+      QString name = tr( "Custom Properties" );
+      QString description = tr( "Variables, custom properties (often used by plugins and custom python code)" );
       switch ( role )
       {
+        case static_cast< int >( Role::NameRole ):
+          return name;
         case Qt::DisplayRole:
-          return tr( "Custom Properties" );
+
+          return htmlStylePattern.arg( name ).arg( description );
         case Qt::ToolTipRole:
-          return tr( "Variables, custom properties (often used by plugins and custom python code)" );
+          return description;
         case Qt::DecorationRole:
           return QgsApplication::getThemeIcon( QStringLiteral( "/mActionOptions.svg" ) );
       }
       break;
+    }
     case QgsMapLayer::StyleCategory::GeometryOptions:
+    {
+      QString name = tr( "Geometry Options" );
+      QString description = tr( "Geometry constraints and validity checks" );
       switch ( role )
       {
+        case static_cast< int >( Role::NameRole ):
+          return name;
         case Qt::DisplayRole:
-          return tr( "Geometry Options" );
+
+          return htmlStylePattern.arg( name ).arg( description );
         case Qt::ToolTipRole:
-          return tr( "Geometry constraints and validity checks" );
+          return description;
         case Qt::DecorationRole:
           return QgsApplication::getThemeIcon( QStringLiteral( "/propertyicons/digitizing.svg" ) );
       }
       break;
+    }
     case QgsMapLayer::StyleCategory::Relations:
+    {
+      QString name = tr( "Relations" );
+      QString description = tr( "Relations with other layers" );
       switch ( role )
       {
+        case static_cast< int >( Role::NameRole ):
+          return name;
         case Qt::DisplayRole:
-          return tr( "Relations" );
+
+          return htmlStylePattern.arg( name ).arg( description );
         case Qt::ToolTipRole:
-          return tr( "Relations with other layers" );
+          return description;
         case Qt::DecorationRole:
           return QgsApplication::getThemeIcon( QStringLiteral( "/propertyicons/relations.svg" ) );
       }
       break;
-
+    }
     case QgsMapLayer::StyleCategory::Temporal:
+    {
+      QString name = tr( "Temporal Properties" );
+      QString description = tr( "Temporal Properties" );
```

Comment:
There's a mix of style here -- should descriptions always be title case? It is here, but not for eg relations. I'd opt for sentence case consistently instead.

Reply:
thanks. I'll do that.